tests/cmake_export/README.md

```diff
@@ -10,4 +10,15 @@ cmake --install build --prefix install
 cd tests/cmake_export
 cmake -B build -DCMAKE_INSTALL_PREFIX=../../install
 cmake --build build
-```
\ No newline at end of file
+```
+
+
+add_subdirectory(ompl)
```

Comment:
Remove this, I found a way to make CMake segfault with an unmatched > in a generator expression and wrote it down.